src/components/crds-site-happenings/site-happenings.tsx

```diff
@@ -332,7 +322,10 @@ export class SiteHappenings {
    */
   private maybeRenderSetSiteModal() {
     if (!this.authToken) return '';
-    if (this.user.site === 'Not site specific' || this.user.site === null || this.user.site === '')
+    if (
+      (this.user.site === 'Not site specific' || this.user.site === null || this.user.site === '') &&
```

Comment:
Can this be simplified to this? null and '' I think are both caught by !this.whatever.
think this can be set to this and be just as meaningful
```suggestion
      (this.user.site === 'Not site specific' || !this.user.site) &&
```